src/image-default.yaml

```diff
@@ -32,3 +32,6 @@ vmware-secure-boot: true
 aws-imdsv2-only: true
 aws-volume-type: "gp3"
 aws-x86-boot-mode: "uefi-preferred"
+
+# Default to not use osbuild
+use-osbuild: false
```

Comment:
Feels like this one is actually more of a `pipeline.yaml` knob https://github.com/coreos/coreos-assembler/pull/3094

but anyways doesn't matter in reality since we'll probably end up taking everything that is in `image.yaml` and moving it in to the mpp yaml after osbuild is the only path?

Reply:
yeah, this one is a bit ugly. I think if we went with a `pipeline.yaml` knob we'd need to add an option (`cosa buildextend-qemu --osbuild`) or env variable to tell COSA what to do.

I'm OK with going that approach too if you think it would be better.

> but anyways doesn't matter in reality since we'll probably end up taking everything that is in `image.yaml` and moving it in to the mpp yaml after osbuild is the only path?

Yeah. We're still trying to figure out the relationship/role of image.yaml and platforms.yaml in the future. I imagine some of this stuff will move around once or twice.

Reply:
Environment variable seems a bit cleaner because in theory this is a change to *how* we build, not *what* we build and we have the `image.yaml`-ends-up-in-commit issue related to that.

Reply:
Changed to env var now.